apko-build/README.md

```diff
@@ -13,6 +13,9 @@ given a config file and base tag to use and output that to a tar file, this does
     config: foo.yaml
     # tag is the base tag to use, required.
     tag: ghcr.io/distroless/foo
+    # use-oci-mediatypes is whether or not to use OCI standard mediatypes.
+    # Optional.
+    use-oci-mediatypes: false
```

Comment:
got confused for a bit.. looks like this action is actually not being used by snapshot, but instead the chainguard one? (https://github.com/chainguard-dev/actions/tree/main/apko-build)